Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: java.lang.RuntimeException: This is a test runtime exception] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: com.example.ProductSpringBoot.Controller.ExceptionController$1: Authentication failed] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: org.springframework.web.client.ResourceAccessException: Resource access denied] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: java.sql.SQLException: SQL error occurred] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: java.util.ConcurrentModificationException: Concurrent modification detected] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: java.util.concurrent.TimeoutException: Request timed out] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: java.util.ConcurrentModificationException: Concurrent modification detected] with root cause
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: MicroUser.UserSpringBoot.Controller.ExceptionController$1: Authentication failed] with root cause
GET /api/exception/exception18
GET /api/exception/exception1
GET /api/exception/exception17
GET /api/exception/exception16
GET /api/exception/exception15
GET /api/exception/exception14
GET /api/exception/exception13
Completed initialization in 3 ms
Initializing Servlet 'dispatcherServlet'
Initializing Spring DispatcherServlet 'dispatcherServlet'
Started ProductSpringBootApplication in 22.459 seconds (process running for 32.466)
Tomcat started on port 8080 (http) with context path '/'
Initialized JPA EntityManagerFactory for persistence unit 'default'
HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
"HHH10001005: Database info:
	Database JDBC URL [Connecting through datasource 'HikariDataSource (HikariPool-1)']
	Database driver: undefined/unknown
	Database version: 17.2
	Autocommit mode: undefined/unknown
	Isolation level: undefined/unknown
	Minimum pool size: undefined/unknown
	Maximum pool size: undefined/unknown"
HikariPool-1 - Start completed.
HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@5710f755
HikariPool-1 - Starting...
No LoadTimeWeaver setup: ignoring JPA class transformer
HHH000026: Second-level cache disabled
HHH000412: Hibernate ORM core version 6.6.2.Final
HHH000204: Processing PersistenceUnitInfo [name: default]
Root WebApplicationContext: initialization completed in 5553 ms
Initializing Spring embedded WebApplicationContext
Starting Servlet engine: [Apache Tomcat/10.1.33]
Starting service [Tomcat]
Tomcat initialized with port 8080 (http)
Finished Spring Data repository scanning in 127 ms. Found 1 JPA repository interface.
Bootstrapping Spring Data JPA repositories in DEFAULT mode.
"No active profile set, falling back to 1 default profile: ""default"""
Starting ProductSpringBootApplication v0.0.1-SNAPSHOT using Java 17.0.13 with PID 1 (/app/app.jar started by root in /app)
GET /api/exception/exception12
GET /api/exception/exception13
GET /api/exception/exception14
GET /api/exception/exception17
GET /api/exception/exception19
Completed initialization in 4 ms
Initializing Servlet 'dispatcherServlet'
Initializing Spring DispatcherServlet 'dispatcherServlet'
Started UserSpringBootApplication in 23.506 seconds (process running for 31.949)
Tomcat started on port 8080 (http) with context path '/'
Initialized JPA EntityManagerFactory for persistence unit 'default'
HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
HikariPool-1 - Start completed.
HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@6a73f164
HikariPool-1 - Starting...
No LoadTimeWeaver setup: ignoring JPA class transformer
HHH000026: Second-level cache disabled
HHH000412: Hibernate ORM core version 6.5.3.Final
HHH000204: Processing PersistenceUnitInfo [name: default]
Root WebApplicationContext: initialization completed in 9859 ms
Initializing Spring embedded WebApplicationContext
Starting Servlet engine: [Apache Tomcat/10.1.31]
Starting service [Tomcat]
Tomcat initialized with port 8080 (http)
Finished Spring Data repository scanning in 1479 ms. Found 2 JPA repository interfaces.
Bootstrapping Spring Data JPA repositories in DEFAULT mode.
"No active profile set, falling back to 1 default profile: ""default"""
Starting UserSpringBootApplication v0.0.1-SNAPSHOT using Java 17.0.13 with PID 1 (/app/app.jar started by root in /app)
"spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning"
Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'Method not supported' is not supported]
"spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning"
"table ""user_model"" does not exist, skipping"
"SQL Warning Code: 0, SQLState: 00000"
"table ""k6model"" does not exist, skipping"
"SQL Warning Code: 0, SQLState: 00000"
"2024-12-16 17:22:21.075 GMT [71] NOTICE:  table ""user_model"" does not exist, skipping"
"2024-12-16 17:22:21.070 GMT [71] NOTICE:  table ""k6model"" does not exist, skipping"
2024-12-16 17:21:58.472 GMT [1] LOG:  database system is ready to accept connections
2024-12-16 17:21:58.448 GMT [66] LOG:  database system was shut down at 2024-12-16 17:21:58 GMT
"2024-12-16 17:21:58.404 GMT [1] LOG:  listening on Unix socket ""/var/run/postgresql/.s.PGSQL.5432"""
"2024-12-16 17:21:58.385 GMT [1] LOG:  listening on IPv6 address ""::"", port 5432"
"2024-12-16 17:21:58.385 GMT [1] LOG:  listening on IPv4 address ""0.0.0.0"", port 5432"
"2024-12-16 17:21:58.385 GMT [1] LOG:  starting PostgreSQL 17.2 (Debian 17.2-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit"
2024-12-16 17:21:58.209 GMT [48] LOG:  database system is shut down
"2024-12-16 17:21:58.201 GMT [50] LOG:  checkpoint complete: wrote 921 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.062 s, sync=0.861 s, total=0.990 s; sync files=301, longest=0.176 s, average=0.003 s; distance=4238 kB, estimate=4238 kB; lsn=0/1908970, redo lsn=0/1908970"
2024-12-16 17:21:57.223 GMT [50] LOG:  checkpoint starting: shutdown immediate
2024-12-16 17:21:57.212 GMT [50] LOG:  shutting down
"2024-12-16 17:21:57.211 GMT [48] LOG:  background worker ""logical replication launcher"" (PID 55) exited with exit code 1"
2024-12-16 17:21:57.208 GMT [48] LOG:  aborting any active transactions
2024-12-16 17:21:56.563 GMT [48] LOG:  received fast shutdown request
2024-12-16 17:21:54.552 GMT [48] LOG:  database system is ready to accept connections
2024-12-16 17:21:54.528 GMT [52] LOG:  database system was shut down at 2024-12-16 17:21:52 GMT
"2024-12-16 17:21:54.465 GMT [48] LOG:  listening on Unix socket ""/var/run/postgresql/.s.PGSQL.5432"""
"2024-12-16 17:21:54.450 GMT [48] LOG:  starting PostgreSQL 17.2 (Debian 17.2-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit"
